float.js

```diff
@@ -475,6 +499,42 @@ const addMarketButtons = function() {
         if (id in floatData) {
             showFloat(id);
         }
+
+        retrieveListingInfoFromPage(id).then(steamListingData => {
+            let listingData = steamListingData[id];
+            if (!listingData) return;
+
+            let assetID = listingData.asset.id;
+            retrieveListingAssets(assetID).then((steamListingAssets) => {
+                const asset = steamListingAssets[assetID];
+                const lastDescription = asset.descriptions[asset.descriptions.length - 1];
+                if (lastDescription.type === 'html' && lastDescription.value.includes('sticker')) {
+                    let imgs = lastDescription.value.replace(/^.*?<center>(.*?)<br>.*?$/g, '$1');
```

Comment:
In order to prevent all of the parsing down below, you can just capture the image groups with regex like `(<img .*?>)` and then just iterate through it like you would with the sticker names.

`lastDescription.value.match(/(<img .*?>)/)`